src/ResourceManager/ServiceBus/Commands.ServiceBus/Utilities/ServiceBusClient.cs

```diff
@@ -828,5 +828,13 @@ public PSCheckNameAvailabilityResultAttributes GetAliasCheckNameAvailability(str
         }
 
         #endregion
+
+        public static ErrorRecord WriteErrorforBadrequest(ErrorResponseException ex)
+        {
+            ErrorResponseContent errorExtract = new ErrorResponseContent();
+            errorExtract = JsonConvert.DeserializeObject<ErrorResponseContent>(ex.Response.Content);
```

Comment:
@v-Ajnava can we do some null-checking in this function to ensure that `ex`, `ex.Response`, and `ex.Response.Content` have a value?